src/plugins/intel_cpu/src/nodes/mkldnn_tensoriterator_node.h

```diff
@@ -129,11 +129,15 @@ class MKLDNNTensorIteratorNode : public MKLDNNNode {
     /* Dynamic support */
     void reshapeSubgraphInput();
     void reshapeAndFillOutput(mkldnn::stream strm);
-    int getNumIteration(const std::vector<PortMap>& inputPortMap, const std::vector<PortMap>& outputPortMap);
+    int getNumIteration(const std::vector<PortMap>& inputPortMap, const std::vector<PortMap>& outputPortMap) const;
+
+    // this method get all memory ptrs of childs of one port to redefine descs for them
+    std::vector<MKLDNNMemoryPtr> getToMemories(const MKLDNNNode* node, const size_t port) const;
```

Comment:
It does not look like a method actually, since it does not access the internal state. M.b. it makes sense to make it internal utility function?